docs/overrides/main.html

```diff
@@ -2,10 +2,10 @@
 
 {% block announce %}
 <div style="color: white;">
-  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2106" style="color: #07e798;">
-    <strong>Prefect 2.10.6 is here!</strong>
-  </a> With multideployment project configuration, improved run restart handling, and
-  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2106" style="color: #07e798;">
+  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2107" style="color: #07e798;">
+    <strong>Prefect 2.10.7 is here!</strong>
+  </a> With a new and improved Flows page, and `on_crashed` state change hook for flows, and
```

Comment:
```suggestion
  </a> With a new and improved flows page, and `on_crashed` state change hook for flows, and
```

For consistency with the release notes lets keep this lower case.

Reply:
The release notes use a capital F, should that be updated, then?

https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#new-and-improved-flows-page

Reply:
If so, I can update it

Reply:
@zhen0 If so, I can update it

Reply:
hmm... the release notes are actually a bit inconsistent now.  They use F in the title but not in the text.  But it's a tiny detail.  Ignore my comment as yours is the best match and follows Bill's original use of F.